openmetadata-ui/src/main/resources/ui/playwright/e2e/Flow/CustomizeLandingPage.spec.ts

```diff
@@ -64,7 +64,7 @@ test.describe('Customize Landing Page Flow', () => {
   test('Add,Remove and Reset widget should work properly', async ({
     adminPage,
   }) => {
-    test.slow(true);
+    test.setTimeout(210000);
```

Comment:
Why do we need the manual timeout?

Reply:
The test timed out in the AUTs and taking more time than when marked slow